src/Panel.js

```diff
@@ -99,6 +99,7 @@ class Panel extends React.Component {
         role={role}
         href={id && `#${id}`}
         onClick={this.handleClickTitle}
+        className={ expanded ? '' : 'collapsed' }
```

Comment:
I just wanted to follow the docs.

Is it an issue to have an empty className? Why?